css/main.css

```
@@ -1,4 +1,4 @@
 body {
   background: #333;
-  color: #fff;
+  color: #321;
```

Po digão, não era cor, era 
```suggestion
  color: #990;
```